static/app/components/dropdownMenuItemV2.tsx

```diff
@@ -0,0 +1,301 @@
+import {
+  ElementType,
+  forwardRef,
+  Fragment,
+  ReactNode,
+  RefObject,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import styled from '@emotion/styled';
+import {useHover, useKeyboard} from '@react-aria/interactions';
+import {useMenuItem} from '@react-aria/menu';
+import {mergeProps} from '@react-aria/utils';
+import {TreeState} from '@react-stately/tree';
+import {Node} from '@react-types/shared';
```

Comment:
hmmm, we're bringing in a _lot_ of spectrum stuff here huh

Reply:
yeah…